static/css/style.css

```diff
@@ -132,16 +139,16 @@ strong{
 }
 
 #frontpage{
-	height: 300px;
-	padding: 5px 5px 20px 5px;
+	height: 350px;
+	background: url(groupmeet.jpg) no-repeat fixed;
```

Comment:
You are referencing the image in your css folder.
To reference the right image (resize it first), use the right path by adding "../" before the image name: url(../groupmeet.jpg)
A quick tip about paths: http://css-tricks.com/quick-reminder-about-file-paths/